e2e/tests/hmctsSuperUserAdministersCase_test.js

```diff
@@ -72,6 +72,30 @@ Scenario('HMCTS super user removes a cmo from a case', async ({I, caseViewPage,
   I.seeInTab([sealedCMO, 'Reason for removal'], 'Entered incorrect order');
 });
 
+Scenario('HMCTS super user removes a sdo from a case', async ({I, caseViewPage, removeOrderEventPage}) => {
```

Comment:
With all these scenarios making new cases thoughts on having each one a separate feature so they can all be run simultaneously? Or have them use the same case, one with all the orders required, so that they aren't having to create a new case each time which just increases flakiness

Reply:
https://getyarn.io/yarn-clip/2c86e1c8-4435-4c2e-b624-1ae6ab784f7a#XQSninGVUz.copy